src/FixedDataTableCell.js

```diff
@@ -197,20 +184,16 @@ class FixedDataTableCell extends React.Component {
     );
 
     var cellProps = {
-      columnKey,
-      height,
-      width,
+      isHeader: this.props.isHeader,
+      isGroupHeader: this.props.isGroupHeader,
+      isFixed: this.props.isFixed,
+      isFixedRight: this.props.isFixedRight,
```

Comment:
Question: Should we expose whether the cell is fixed/fixedRight through 2 props? or with something like a single `cellGroupType` enum prop, which can be directly used with the public API methods?

eg: 
```
if (this.props.isFixed) {
  return this.context.getCellGroupWidth('fixed')
} else if (this.props.isFixedRight) {
  return this.context.getCellGroupWidth('fixedRight');
} else {
  return this.context.getCellGroupWidth('scrollable');
}
```
vs
```
  return this.context.getCellGroupWidth(this.props.cellGroupType);
```